app/classes/inat/inat_constants.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
```

Comment:
@nimmolo, @mo-nathan Does nesting this PORO under Inat make sense?

Reply:
Seems reasonable to me.  What I would look at is whether things are largely self contained between the classes within the `inat` directory.  If there's an objects that is getting used in many different places or for different purposes, then it might make sense to move it somewhere else.  In the abstract it would be better if this was `app/classes/inat/constants.rb`, but I'm not sure off the top of my head if that works well in Ruby given the autoloading magic.